astroquery/mast/collections.py

```diff
@@ -40,6 +40,10 @@ def __init__(self):
 
         super().__init__()
 
+        services = {"panstarrs": {"path": "panstarrs/{data_release}/{table}.json",
+                                  "args": {"data_release": "dr2", "table": "mean"}}}
+        self._service_api_connection._set_service_params(services, "catalogs", True)
```

Comment:
The `ServiceAPI` class seems to have all private accessors (`_set_service_params()`, `_SERVICES`, etc.).  Are they meant to be public, and just need a name change, or are they actually private and not meant for the external access they're getting?

Reply:
Will handle in future PR.